src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: flex-end;
+}
+
+.logo {
+  margin-left: 50px;
+  margin-right: auto;
+  align-self: center;
+}
+
+.logo img {
+  display: block;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav_list_group {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0 10px;
+  position: relative;
+}
+
+.nav_list:first-child {
+  margin-left: 0;
+}
+
+.nav_list:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav_link:hover {
+  color: #00acdc;
```

Comment:
Move repeatable colors into variables

Reply:
Not fixed. It would be great to create variables for all repeatable values

Reply:
Not fixed
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review